### DISCUSSION TITLE
Loco Positioning and Fake GPS

### DISCUSSION THREAD

**Discussion by hihihellohihi** [bitcraze/discussions#1397] [Q&A] [answered]
Hello, is it possible to use the Loco Positioning anchors/tags to give a fake GPS coordinate to an ArduPilot flight controller?

**Comment by knmcguire** [comment, marked as answer]
Hi! We don't have much experience with the ardupilot flight controller but we are sure that it could be possible.

Here are some discussions of people trying to do the same for PX4:
- https://github.com/orgs/bitcraze/discussions/1272
- https://github.com/orgs/bitcraze/discussions/1019
- https://github.com/orgs/bitcraze/discussions/775

**Reply by hihihellohihi** [reply]
Thanks! I'll try these out
